packages/muelu/adapters/xpetra/MueLu_RefMaxwell_def.hpp

```diff
@@ -204,8 +204,8 @@ namespace MueLu {
     TEUCHOS_ASSERT(dirichletDomain.extent(0) == domMap->getNodeNumElements());
     RCP<Xpetra::Vector<Scalar,LocalOrdinal,GlobalOrdinal,Node> > myColsToZero = Xpetra::VectorFactory<Scalar,LocalOrdinal,GlobalOrdinal,Node>::Build(colMap, /*zeroOut=*/true);
     // Find all local column indices that are in Dirichlet rows, record in myColsToZero as 1.0
-    auto myColsToZeroView = myColsToZero->getDeviceLocalView();
-    auto localMatrix = A.getLocalMatrix();
+    auto myColsToZeroView = myColsToZero->getDeviceLocalView(Xpetra::Access::ReadWrite);
```

Comment:
Wouldn't write-only be enough here?

Reply:
Same as below, i am not sure that myColsToZero are 100% overwritten, can you confirm that it is.  I don't believe we can guarantee that initial values are 0